srcpkgs/vaultwarden-web/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'vaultwarden-web'
+pkgname=vaultwarden-web
+version=2.28.1
+revision=1
+_patch_version=2.28.0
+create_wrksrc=yes
+hostmakedepends="git nodejs"
+short_desc="Web vault builds for vaultwarden"
+maintainer="Joel Beckmeyer <joel@beckmeyer.us>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/dani-garcia/bw_web_builds"
+distfiles="https://raw.githubusercontent.com/dani-garcia/bw_web_builds/v${version}/patches/v${_patch_version}.patch"
+checksum=5abd6c839f2a2ed340b46bd1a0ecde8c71adba12bc72e97e69832077b639e874
+
+post_fetch() {
+	# need to directly clone and checkout version so we can pull in submodules
+	git clone https://github.com/bitwarden/web ${wrksrc}
+	cd ${wrksrc}
+	git checkout "v${version}"
+	git submodule update --init
+}
+
+post_patch() {
+	patch --forward --strip=1 --input="v${_patch_version}.patch"
```

Comment:
You mean like the patches included in `srcpkgs/$package/patches`? Yes, it can be applied with the same args to patch (`patch -s -Np1 < v2.28.0.patch`), but we can't just do `mv v2.28.0.patch $PATCHESDIR`. What do you have in mind here?

Reply:
I was thinking just vendor it